### DISCUSSION TITLE
What do the subfolders inside the chunks folder correspond to ?

### DISCUSSION THREAD

**Discussion by shodanx2** [mediacms-io/mediacms#579] [Q&A] [answered]
I am running out of space and so I'm checking out various folder for unecessary files

I have gotten rid of the originals and I see my videos are stored in the encoded folder.

So what exactly are the folders and files in the chunks folder ? 

![image](https://user-images.githubusercontent.com/10621885/203160084-d43d8c42-e5ed-4db8-b750-0387ff5a3022.png)

Are these remains of failed encodings ? Or are they part of an actual served media file ?

![image](https://user-images.githubusercontent.com/10621885/203160288-0f044684-0b5d-4236-9301-9da2d726e2cc.png)

**Comment by mgogoulos** [comment, marked as answer]
Hi, you are correct, they are failed encodings / temporary files that were not deleted. There should be a command (or scheduled task) to check and remove them from time to time, but for now you may delete them manually

**Reply by shodanx2** [reply]
Thank you !
